my-app/src/services/employees.service.js

```diff
@@ -0,0 +1,62 @@
+import Constants from "../constants/constants.service";
+import authService from "./auth.service";
+import departmentsService from "./departments.service";
+
+const baseUrl = Constants.getBaseUrl();
+
+const getEmployees = () => {
+  const options = {
+    method: "GET",
+    headers: {
+      authorization: authService.getToken(),
+    },
+  };
+
+  return fetch(`${baseUrl}/employees`, options).then(async (res) => {
+    if (!res.ok) {
+      throw new Error(res.statusText);
+    }
+
+    const data = await res.json();
+    return data?.employees || [];
+  });
+};
+
+const getEmployeesByDepartmentId = (departmentId) => {
+  const options = {
```

Comment:
this object is used in function above, why would not you move it beyond the the functions and use just reference?

Reply:
done